.gitignore

```diff
@@ -69,7 +69,7 @@ typings/
 .yarn-integrity
 
 # dotenv environment variables file
-.env
```

Comment:
We don't want to remove the .env from the .gitignore file. Instead, put everything in the .env file in .env.sample.